taxonomies/ingredients_processing.txt

```diff
@@ -601,7 +612,7 @@ en:pasteurised, pasteurized
 ca:pasteuritzada
 da:pasteuriseret
 de:pasteurisiert, pasteurisierte, pasteurisierter
-es:pasteurizado, pasteurizada, pasteurizados, pasteurizadas, producto pasteurizado, producto pasteurizada
+es:pasteurizado, pasteurizada, pasteurizados, pasteurizadas, producto pasteurizado, producto pasteurizada,pasterizada
```

Comment:
is "pasterizada" a mispelling? if it is, we should not add it. We are going to use taxonomies to automatically correct the spelling of ingredients lists, but for that to work, they need to be free of mispellings.